src/Sylius/Component/Shipping/Model/ShippingMethod.php

```diff
@@ -230,6 +237,54 @@ public function removeRule(RuleInterface $rule)
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function addChannel(Channel $channel)
```

Comment:
change `Channel` to `ChannelInterface` both here and in the interface then I'll merge